src/components/counters/Counter.tsx

```diff
@@ -83,21 +141,30 @@ const Counter = ({
   children,
   className,
   size,
-  color = 'red-60',
+  color = 'red',
+  type = 'solid',
```

Comment:
sorry Dawid, i somehow missed it, could you change `type` to `variant` we are migrating to the new name since `type` is reserved name used often in html (e.g. label will be changed, button already is)

Reply:
sure thing @coderitual ! I changed that, hope it's fine now